packages/@magic-ext/oauth2/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@magic-ext/oauth2",
```

Comment:
OAuth2 is little ambiguous, as it might indicate OAuth 2 protocol that has access token. 

What do you think `oauth-v2`?

Reply:
My thought here was intentionally have it indicate both v2 and oauth 2.0 standard. That way when the legacy `oauth` package goes away, this new package name will still make sense

Reply:
This also help consistency with npm package name and the SDK property being the same: `getMagicInstance().oauth2.loginWithRedirect({
        provider,
        redirectURI,
      });`